### PR TITLE
Feedback Source

### PR DESCRIPTION
**Pull request by rrpff**

### CODE REVIEW

**Review by Swiftwork — APPROVED** [review]
Out of curiosity and education, can you write a brief comment description for when we can expect each source?

```
"api" | "manual" | "prompt" | "sdk" | "widget"
```

File: src/types.ts [hunk 1]
```diff
@@ -138,6 +138,11 @@ export type Feedback = {
    * feedback.
    */
   promptId?: FeedbackPrompt["promptId"];
+
+  /**
+   * Source, depending on how the user was asked
+   */
+  source?: "api" | "manual" | "prompt" | "sdk" | "widget";
```
Comment (pavkam) [inline src/types.ts:145]: Should we remove `api` and `manual` from this enum? The SDK does not use them and probably better not to be advertised.

Comment (rrpff) [reply src/types.ts:145]: Yep, good shout

**Comment by rrpff** [comment]
@Swiftwork good shout - added some extra info to the comment. I don't expect this property to be used by customers, but didn't want to introduce extra misdirection to avoid it being visible.

**Pull request merged**